src/main/java/seedu/moneygowhere/storage/LocalStorage.java

```diff
@@ -1,4 +1,115 @@
 package seedu.moneygowhere.storage;
 
+import seedu.moneygowhere.common.Messages;
+import seedu.moneygowhere.data.expense.Expense;
+import seedu.moneygowhere.data.expense.ExpenseManager;
+import seedu.moneygowhere.exceptions.LocalStorageLoadDataInputError;
+
+import static seedu.moneygowhere.common.Configurations.DIRECTORY_PATH;
+import static seedu.moneygowhere.common.Configurations.FILE_PATH_EXPENSES;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class LocalStorage {
+    private static final String DIVIDER = " // ";
```

Comment:
This divider string may cause issues if the expense's name, description or category contains the same string. Do consider creating a function to encode and decode this divider when saving and loading expenses.